config/config.go

```diff
@@ -0,0 +1,138 @@
+package config
+
+import (
+	"fmt"
+	"github.com/Portshift/klar/docker"
+	"github.com/Portshift/klar/utils"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+//Used to represent the structure of the whitelist YAML file
+type vulnerabilitiesWhitelistYAML struct {
+	General []string
+	Images  map[string][]string
+}
+
+const (
+	OptionClairOutput        = "CLAIR_OUTPUT"
+	OptionClairAddress       = "CLAIR_ADDR"
+	OptionKlarTrace          = "KLAR_TRACE"
+	OptionClairThreshold     = "CLAIR_THRESHOLD"
+	OptionClairTimeout       = "CLAIR_TIMEOUT"
+	OptionDockerTimeout      = "DOCKER_TIMEOUT"
+	OptionJSONOutput         = "JSON_OUTPUT" // deprecate?
+	OptionFormatOutput       = "FORMAT_OUTPUT"
```

Comment:
not used

Reply:
This is part of the current klar (to be used from outside to print). I just moved the file